app/backplane/sensor_module/src/c_sensing_tenant.cpp

```diff
@@ -85,9 +85,35 @@ void CSensingTenant::Run() {
 
     data.Temperature.Temperature = thermometer.GetSensorValueFloat(SENSOR_CHAN_AMBIENT_TEMP);
 
-    detectionHandler.HandleData(uptime, data, sensor_states);
     // If we can't send immediately, drop the packet
     // we're gonna sleep then give it new data anywas
     dataToBroadcast.Send(data, K_NO_WAIT);
-    dataToLog.Send(data, K_NO_WAIT);
+    sendDownlinkData(data);
+
+    detectionHandler.HandleData(uptime, data, sensor_states);
+    if (detectionHandler.FlightOccurring()) {
+        dataToLog.Send(data, K_NO_WAIT);
+    }
+}
+
+void CSensingTenant::sendDownlinkData(const NTypes::SensorData& data) {
+    NTypes::LoRaBroadcastSensorData downlinkData{
+        .Barometer = {
+            .Pressure = static_cast<int16_t>(data.PrimaryBarometer.Pressure),
+            .Temperature = static_cast<int16_t>(data.PrimaryBarometer.Temperature),
+        },
+        .Acceleration = {
+            .X = static_cast<int16_t>(data.Acceleration.X),
```

Comment:
do you wwant to multiply these things? idrk what rad/s to expect from gyro but for acceleration you're losing a lot of space since for most of the time youll be seeing 9.8 m/s^2 maybe up to 200 m/s^2 if we're really cooking at 20g. thats like 0.6% of the full scale of the int16 if im doing my math right

Reply:
it looks like u added the getMilliUnits for something like this but arent using them here?

Reply:
I think its a decent trade-off between optimization and QOL. We avoid extra processing doing the conversion in both FSW and GSW at the expense of like 3 bytes. In comparison to what we had previously, I think this is fine. The getMilliUnits functions were more for Zephyr reporting at a scale that is too large to be captured with a fixed point. For example, drawing less than an amp would just report 0 in fixed point.

Reply:
If we were really stretched, then yes we could do it. I think its important to consider when we are over-optimizing.

Reply:
I sorta see your point but i dont feel like multiplying by 10 or 100 is really that big a trade off. Right now we're throwing away a bunch of data and still sending 16 bits. We could throw away less data and still send the same 16 bits. 
or even drop it to 8 - at least that way we're throwing away precision but actually getting packet size benefits out of it rather than losing precision for no gain. 

for something that happens as infrequently as data downlink, multiplying by 10 or 100 really isn't that big an issue

Reply:
Its not really a performance consideration. Its just trying to limit the amount of agreement we need to do between two different codebases. Its still going to multiply every time though because its being handled by the sensor module itself to keep things decoupled from radio module

Reply:
I hardly think not being wasteful is over optimization esp when the whole point of this PR is to make the packets more information dense - thats kinda the MO of the feature